xbmc/cores/paplayer/PAPlayer.cpp

```diff
@@ -312,7 +312,23 @@ bool PAPlayer::QueueNextFileEx(const CFileItem &file, bool fadeIn)
 
   StreamInfo *si = new StreamInfo();
   si->m_fileItem = file;
-  if (!si->m_decoder.Create(file, si->m_fileItem.m_lStartOffset))
+
+  // Start stream at zero offset
+  si->m_startOffset = 0;
+  //File item start offset defines where in song to resume
+  double starttime = CUtil::ConvertMilliSecsToSecs(si->m_fileItem.m_lStartOffset);
+
+  // Music from cuesheet => "item_start" and offset match
+  // Start offset defines where this song starts in file of multiple songs
+  if (si->m_fileItem.HasProperty("item_start") &&
+      (si->m_fileItem.GetProperty("item_start").asInteger() == si->m_fileItem.m_lStartOffset))
+  {
+    // Start stream at offset from cuesheet
+    si->m_startOffset = si->m_fileItem.m_lStartOffset;
+    starttime = 0; // No resume point
+  }
```

Comment:
Maybe it makes sense to make the `double starttime = CUtil::ConvertMilliSecsToSecs(si->m_fileItem.m_lStartOffset);` into an `else`, as by `if(true)` it becomes changed to "0"?

Reply:
I implelented the logic in this way around to highlight that the cuesheet use of offset is the minority use.